packages/core/src/graph/controller/item.ts

```diff
@@ -478,6 +498,12 @@ export default class ItemController {
         graph.showItem(c, stack);
       }
     });
+    
+    if (typeof combo !== 'string') {
+      setTimeout(() => {
+        this.updateParentCombo(combo as ICombo);
+      }, 200);
```

Comment:
There's some race condition happening here requiring this action to be delayed

https://user-images.githubusercontent.com/15673201/194109893-7b981b0e-0271-4246-9f4d-3a776f2e726a.mov